data/transactions/logic/README.md

```diff
@@ -139,6 +139,9 @@ For three-argument ops, `A` is the element two below the top, `B` is the penulti
 | `getbyte` | pop a byte-array A and integer B. Extract the Bth byte of A and push it as an integer |
 | `setbyte` | pop a byte-array A, integer B, and small integer C (between 0..255). Set the Bth byte of A to C, and push the result |
 | `concat` | pop two byte-arrays A and B and join them, push the result |
+| `ecdsa_verify c` | for (data A, signature B, C and pubkey D, E) verify the signature of the data against the pubkey => {0 or 1} |
```

Comment:
This is confusing because c is the name of the field (the immediate) and also (I think) the name you're using to describe the stack value.  I'd pick a new name for the immediate.

Also need to say what the immediate is/does.